Cataclysm/DeathKnight.lua

```diff
@@ -6,6 +6,11 @@ local class, state = Hekili.Class, Hekili.State
 
 local spec = Hekili:NewSpecialization( 6 )
 
+-- TODO
+--- Unholy Presence reduces global cooldown by .5 seconds. (Unsure how to do this)
```

Comment:
It'll be in State.lua, see where adrenaline_rush and voidform(?) are referenced.